includes/class-sensei-admin.php

```diff
@@ -109,7 +109,7 @@ public function admin_menu() {
 		}
 
 		add_submenu_page( 'edit.php?post_type=course', __( 'Order Courses', 'sensei-lms' ), __( 'Order Courses', 'sensei-lms' ), 'manage_sensei', $this->course_order_page_slug, array( $this, 'course_order_screen' ) );
-		add_submenu_page( 'edit.php?post_type=lesson', __( 'Order Lessons', 'sensei-lms' ), __( 'Order Lessons', 'sensei-lms' ), 'edit_lessons', $this->lesson_order_page_slug, array( $this, 'lesson_order_screen' ) );
+		add_submenu_page( 'edit.php?post_type=lesson', __( 'Order Lessons', 'sensei-lms' ), __( 'Order Lessons', 'sensei-lms' ), 'manage_sensei_grades', $this->lesson_order_page_slug, array( $this, 'lesson_order_screen' ) );
```

Comment:
It would be best to stay close to the lesson's capabilities here because this is lesson-related.

I suggest that we use the `edit_published_lessons`, which should be available for Teachers, Authors, and above. 🙂

Reply:
It kinda makes sense for authors to be able to order lessons because they can create/publish them, but they can edit only their own courses. So I think it makes sense for the order form to only list courses created by that author (similar to how it works for teachers). Toughts?

Reply:
Makes complete sense!